addons/mail/models/res_users.py

```diff
@@ -24,6 +25,8 @@ class ResUsers(models.Model):
         help="Policy on how to handle Chatter notifications:\n"
              "- Handle by Emails: notifications are sent to your email address\n"
              "- Handle in Odoo: notifications appear in your Odoo Inbox")
+    presence_id = fields.One2many("mail.presence", "user_id")
```

Comment:
one2many should be suffixed plural `_ids` (even if there is only one in practice)